_addons_transformations/scale/readme.md

```diff
@@ -14,17 +14,50 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
+The input string must be in numerical format.
 
-The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
+The file is expected to exist in the `transform` configuration directory and its ending has to be `.scale`.
+It should follow the format given in the table below.
+
+Range expressions always contain two parts.
+The range to scale on, which is located left from the equality sign and the corresponding output string on the right of it.
+A range consists of two bounds. Both are optional, the range is then open. Both bounds can be inclusive or exclusive.
+
+| Scale Expression | Returns XYZ when the given Value is                        |
+|------------------|------------------------------------------------------------|
+| `[12..23]=XYZ`   | `between (or equal to) 12 and 23`                          |
+| `]12..23[=XYZ`   | `between 12 and 23 (12 and 23 are excluded in this case.)` |
+| `[..23]=XYZ`     | `lower than or equal to 23`                                |
+| `]12..]=XYZ`     | `greater than 12`                                          |
 
 ## Example
 
-transform/temps.scale:
+The following example shows how to break down numeric UV values into fixed UV index categories.
+We have an example UV sensor that sends us numeric values from `0` to `100`, which we then want to scale into the *UV Index* range.
+For further Information on the UV Index take a look at [Wikipedia](https://en.wikipedia.org/wiki/Ultraviolet_index).
+
+Example item:
+
+```java
+Number Uv_Sensor_Level "UV Level [SCALE(uvindex.scale):%s]"
+```
+
+Uvindex.scale in transform folder:
 
-```properties
-[12..23]=Temp between 12 or higher and 23 or lower
-]12..23[=Temp over 12 and below 23
-..23]=Any temp up to and including 23
-]12..=Any temp above 12
+```python
+[..3]=1
+]3..6]=2
+]6..8]=3
+]8..10]=4
+[10..100]=5
```

Comment:
Leave right-unbounded? (remove "100")

Reply:
In fact i would leave it this way.
With this setup we have an example on which we can explain what happens with values which are not recognized by any of the matching ranges.